addon/mixins/in-viewport.js

```diff
@@ -63,7 +65,7 @@ export default Mixin.create({
     const owner = getOwner(this);
 
     if (owner) {
-      return merge(defaultOptions, owner.lookup('config:in-viewport'));
+      return assign(defaultOptions, owner.lookup('config:in-viewport'));
```

Comment:
Should be:

    return assign({}, defaultOptions, owner.lookup('config:in-viewport'));